flutter/lib/src/event_processor/android_platform_exception_event_processor.dart

```diff
@@ -152,14 +185,36 @@ extension on JvmException {
       return entry.value.toSentryStackFrame(entry.key, nativePackageName);
     }).toList(growable: false);
 
-    return SentryException(
+    var exception = SentryException(
       value: description,
       type: exceptionType,
       module: module,
       stackTrace: SentryStackTrace(
         frames: stackFrames.reversed.toList(growable: false),
       ),
     );
+
+    String threadName;
+    if (thread != null) {
+      // Needs to be prefixed with 'Android', otherwise this thread id might
+      // clash with isolate names from the Dart side.
+      threadName = 'Android: $thread';
+    } else {
+      // If there's no thread in the exception, we just indicate that it's
+      // from Android
+      threadName = 'Android';
+    }
+    final threadId = threadName.hashCode;
+
+    final sentryThread = SentryThread(
+      crashed: true,
+      current: true,
```

Comment:
This is also up for debate: The Dart isolate is active, so the Android thread isn't active (`current`). So maybe that should be as follows? 
```suggestion
      current: false,
```